servicetalk-examples/grpc/helloworld-maven/src/main/java/io/servicetalk/examples/grpc/helloworld/BlockingHelloWorldClient.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2018 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.examples.grpc.helloworld.blocking;
+
+import io.servicetalk.grpc.netty.GrpcClients;
+
+import io.grpc.examples.helloworld.Greeter.BlockingGreeterClient;
+import io.grpc.examples.helloworld.Greeter.ClientFactory;
+import io.grpc.examples.helloworld.HelloReply;
+import io.grpc.examples.helloworld.HelloRequest;
+
+public final class BlockingHelloWorldClient {
```

Comment:
Also add a server so the example is self sufficient?

Reply:
I just moved the `pom.xml` into the existing `helloworld` example to avoid duplication